field-value-counter-sink/src/main/java/org/springframework/cloud/stream/module/metrics/FieldValueCounterSinkProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.metrics;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+@ConfigurationProperties
+public class FieldValueCounterSinkProperties extends MetricProperties {
+
+	private String fieldName;
+
+	public String getFieldName() {
```

Comment:
Also, not necessarily part of this PR, but ability to store / increment multiple fields at once should be considered